Tests/LinuxMain.swift

```diff
@@ -0,0 +1,7 @@
+import XCTest
```

Comment:
Does this need to go in the `SwishTests` directory?

Reply:
I don't think this file even needs to exist, since we're only supporting Apple platforms at the moment